oluseyianani/React-App-Boilerplate
# #165333813 Create Webpack Intial configuration

### What does this PR do?
- creates webpack initial configuration

### Description of Task to be completed?
- `webpack.config.js` setup
- adds `.gitignore` file

### How should this be manually tested?
- run  `npm run dev` on this branch
- on your console, you'll see a log `BOOM!!!!! REACT APP BOILERPLATE BABY`

### Any background context you want to provide?
n/a

### What are the relevant pivotal tracker stories?
[#165333813](https://www.pivotaltracker.com/story/show/165333813)

### Screenshots (if appropriate)
<img width="1440" alt="Screenshot 2019-04-14 at 12 21 36 PM" src="https://user-images.githubusercontent.com/25327188/56092135-dfe31600-5eaf-11e9-884a-c658d7919873.png">
